tests/test_workdir.py

```diff
@@ -57,12 +59,53 @@ def test_directory_structure_generated(self, spatial_directory_list):
             assert path.exists()
 
 
-class TestCleanDirectoryTree:
-    """Tests for `clean_directory_tree()`."""
+class TestCleanFiles:
+    """Tests for `clean_directory_tree()` and `clean_cwd_logfiles)."""
 
-    @pytest.mark.parametrize("test_path", [Path("runs"), Path("src")])
-    def test_clean_directory_tree(self, test_path):
-        """Success case: directory tree does not exist after clean."""
-        test_path.mkdir()
-        clean_directory_tree()
-        assert not test_path.exists()
+    @pytest.fixture(autouse=True)
+    def set_internal_cwd(self, monkeypatch):
+        """Sets internal.CWD to pytest's working directory."""
+        monkeypatch.setattr(internal, "CWD", Path.cwd())
```

Comment:
I don't think we need to do that because `CWD` in internal is set by running `Path.cwd()`.

Reply:
`internal.CWD` is indeed being set by benchcab, however is is at the start of the program. By the time `pytest` runs `test_workdir`, `Path.cwd()` changes to a temporary directory created by pytest, so we need to update `internal.CWD` as well (this is being used by `clean_realisation_files`:

Before running `set_internal_cwd`: `/home/659/ag9761/ACCESS-NRI/benchcab`
After running `set_internal_cwd` sample: `/scratch/tm70/ag9761/tmp/benchcab_testsxn_1igdj`

Reply:
And we want to have cwd in the /scratch/ directory only, not conflicting in `benchcab` root